submissions/SofiiaTrokhymchuk/tiny-js-world/index.js

```diff
@@ -0,0 +1,103 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/SofiiaTrokhymchuk/a-tiny-JS-world
+   Web app: https://sofiiatrokhymchuk.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const dog = {
+   species: 'dog',
+   name: 'Bima',
+   gender: 'female',
+   legs: 4,
+   hands: 0,
+   saying: 'Bark! ▼・ᴥ・▼'
+};
+
+const cat = {
+   species: 'cat',
+   name: 'Jessy',
+   gender: 'female',
+   legs: 4,
+   hands: 0,
+   saying: 'Meow! (=^･ω･^=)'
+};
+
+const male = {
+   species: 'human',
+   name: 'Ash Lynx',
+   gender: 'male',
+   legs: 2,
+   hands: 2,
+   saying: 'Stay with me... I won\'t ask "forever." Just for now, Eiji.'
+};
+
+const female = {
+   species: 'human',
+   name: 'Temari',
+   gender: 'female',
+   legs: 2,
+   hands: 2,
+   saying: 'Amazing. This slacker-clown outsmarted me!'
+};
+
+const catWoman = Object.create(cat);
+catWoman.species = 'cat-woman';
+catWoman.name = 'Hello Kitty';
+catWoman.legs = 2;
+catWoman.hands = 2;
+
+function inhabitantToString(inhabitant){
+   const friends = [];
+   if(inhabitant.friends === undefined){
+      friends.push(0);
+   }else{
+      friends.push(...inhabitant.friends);
+   }
```

Comment:
1) Can we employ ternary operator here?
2) `friends.push(0)`: why not assign a value of `[0]`?
3) `friends.push(...inhabitant.friends)`: `arr = [...sourceArr]` is another (ES6) way to copy elements into a new array. But why do we need to have a copy at all?